assets/src/edit-story/elements/media/util.js

```diff
@@ -98,4 +98,15 @@ function getThumbnailUrl(minWidth, resource) {
   return resource.src;
 }
 
-export default getThumbnailUrl;
+/**
+ * Choose the preview video URL if available, or else choose the normal source URL.
+ *
+ * @param {*} resource Image resource object.
+ * @return {string} Source URL of the smallest available size image.
+ */
+export function getPreviewVideoUrl(resource) {
+  if (resource.sizes && resource.sizes.preview) {
+    return resource.sizes.preview.source_url;
+  }
+  return resource.src;
+}
```

Comment:
Even though this works, can't we reuse the logic in getThumbnailUrl that returns the smalles `sizes` largest than a given width?

Reply:
I don't think `preview` actually provides a `width` attribute, only `full` does.

Reply:
After https://github.com/google/web-stories-wp/pull/3893 we will provide sizes

Reply:
Done.